cli/assets/.gitignoreTemplate

```diff
@@ -6,6 +6,7 @@ out/
 !/broadcast
 /broadcast/*
```

Comment:
Should this be removed from the default template now?

Reply:
resolved